libknet/bindings/rust/src/sys/libknet.rs

```diff
@@ -0,0 +1,2544 @@
+/* automatically generated by rust-bindgen 0.57.0 */
```

Comment:
if there are files that are automatically generated, should we just generate them at build time vs shipping them?